### PR TITLE
Added translation using Weblate (Turkish)

### PR DESCRIPTION
**Pull request by yagizhanNY**
Hi, I finished the Turkish translation. Could you check? If any feedback, please let me know.

### CODE REVIEW

**Comment by WofWca** [comment]
I'll review on detail tomorrow, but for now:
1. Did you mean to specify a different commit author? It doesn't appear to be associated with your GitHub account.
2. Just curious, why did you not use Weblate and why does the commit message say "translated using Weblate"?
3. Is this for 'tr' language, not 'ota'? I saw 'ota' was also recently added on Weblate.

**Comment by yagizhanNY** [comment]
1. Yes name is different but it is my GitHub account.
2. It's my bad. I didn't know how Weblate works. I checked the previous commits' messages. If it is important for you I can change the message and commit again.
3. I tried to learn how Weblate works. While that I guess I added 'ota' language. Could you delete that? Yes, this is just for tr language.

File: tr/messages.json [hunk 1]
```diff
@@ -1 +1,430 @@
...
+		"message": "ses"
+	},
+	"video": {
+		"message": "video"
+	},
+	"contentScriptFail": {
+		"message": "Bu sayfadaki içerik komut dosyası yürütülemedi"
+	},
+	"suggestOpenLocalFile": {
+		"message": "**$openLocalFile$**?mak mı istiyorsunuz?",
```
Comment (WofWca) [inline tr/messages.json:84]: Does this look right? 
image

File: tr/messages.json [hunk 2]
```diff
@@ -1 +1,430 @@
-{}
+{
+	"extensionDescription": {
+		"message": "Cümleler arasındaki uzun duraksamaları ileri sararak ~ 1,5 kat daha hızlı izleyin."
```
Comment (WofWca) [inline tr/messages.json:3]: Is the comma good here?

**Review by WofWca — CHANGES_REQUESTED** [review]
I've merged so it's shown on Weblate, we'll amend if needed. You may edit stuff there, especially mismatched commas (if required): https://hosted.weblate.org/projects/jump-cutter/extension-ui/tr/

Here's also a screenshot of the options page:
image

File: tr/messages.json [hunk 3]
```diff
@@ -1 +1,430 @@
...
+		"message": "Medyanın sesi kapatılırsa sayfayı yenileyin"
+	},
+	"volumeThreshold": {
+		"message": "En düşük konuşma yüksekliği"
+	},
+	"volumeThresholdTooltip": {
+		"message": "Mevcut ses seviyesi bu değerin altına düştüğünde, hızlandır.\nGrafikte, bu kırmızı 🟥 yatay çizgidir. Mavi 🟦 çizgi mevcut ses seviyesidir, bu nedenle mavi çizgi kırmızının altında olduğunda sessizlik olarak kabul edilir."
+	},
+	"soundedSpeed": {
+		"message": "Sounded speed"
```
[the remark this answers is not in the document]
Comment (yagizhanNY) [reply tr/messages.json:138]: Ohh sorry. Yes I missed that. I will change and commit again for this.

File: tr/messages.json [hunk 4]
```diff
@@ -1 +1,430 @@
...
+		"message": "**$openLocalFile$**?mak mı istiyorsunuz?",
+		"placeholders": {
+			"openLocalFile": {
+				"content": "$1",
+				"example": "Yerel bir dosya aç"
+			}
+		}
+	},
+	"openLocalFile": {
+		"message": "Yerel bir dosya aç"
```
[the remark this answers is not in the document]
Comment (yagizhanNY) [reply tr/messages.json:93]: It is possible. I checked this, we have just one problem about this. You already mention that below. I am going to fix it.

File: tr/messages.json (hunk 2 above)
Comment (WofWca) [reply tr/messages.json:3]: I mean, isn't it gonna look a little weird? There is no comma in any other translation.

Comment (yagizhanNY) [reply tr/messages.json:3]: In Turkish it must be comma. You can see an example for different cultures from Microsoft's document below.

image

Comment (WofWca) [reply tr/messages.json:3]: Oh my god I mixed up "comma" with "period". I meant the period at the end of the string.

Comment (WofWca) [reply tr/messages.json:3]: Alright, I'll remove it for now, you can put it back if it's more correct.

Comment (yagizhanNY) [reply tr/messages.json:3]: You can remove it doesn't mater.

**Comment by WofWca** [comment]
Major thanks!
Coming in `1.24.6`, gonna submit now.

**Pull request merged**